values.schema.json

```diff
@@ -0,0 +1,479 @@
+{
```

Comment:
Either all fields should have default+descriptions set, or none should IMO. I'm leaning towards not having defaults+descriptions set and relying on the values file in the repo on being this source of info

Reply:
or generating values.yaml from this